cfme/tests/openstack/infrastructure/test_hosts.py

```diff
@@ -115,3 +127,16 @@ def test_host_zones_assigned(provider):
         host = Host(name=quad.name, provider=provider)
         result = host.get_detail('Relationships', 'Availability Zone')
         assert result, "Availability zone doesn't specified"
+
+
+@pytest.mark.trylast
+def test_host_delete(provider):
+    navigate_to(provider, 'ProviderNodes')
+    quads = filter(lambda q: 'Compute' in q.name, Quadicon.all())
+    hname = random.choice(quads).name
```

Comment:
**OPTIONAL** in future it's probably best to put the full ```hostname``` it's not that much longer in length